modules/nw-ipfailover-remove.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ipfailover.adoc
+
+[id="nw-ipfailover-remove_{context}"]
+= Removing IP failover
+
+When IP failover is initially configured, the worker nodes in the cluster are modified with an `iptables` rule that explicitly allows multicast packets on `224.0.0.18` for Keepalived. Because of the change to the nodes, removing IP failover requires running a job to remove the `iptables` rule.
+
+.Procedure
+
+. Optional: Identify and delete any check and notify scripts that are stored as config maps.
+
+.. Identify whether any pods for IP failover use a config map as a volume:
++
+[source,terminal]
+----
+$ oc get pod -l ipfailover -A \
+  -o jsonpath="\
+{range .items[?(@.spec.volumes[*].configMap)]}
+{'Namespace: '}{.metadata.namespace}
+{'Pod:       '}{.metadata.name}
+{'Volumes that use config maps:'}
+{range .spec.volumes[?(@.configMap)]}  {'volume:    '}{.name}
+  {'configMap: '}{.configMap.name}{'\n'}{end}
+{end}"
+----
++
+.Example output
+----
+Namespace: default
+Pod:       keepalived-worker-59df45db9c-2x9mn
+Volumes that use config maps:
+  volume:    config-volume
+  configMap: mycustomcheck
+----
+
+.. If the preceding step provided the names of config maps that are used as volumes, delete the config maps:
++
+[source,terminal]
+----
+$ oc delete configmap -n <namespace> <configmap_name>
+----
+
+. Identify an existing deployment for IP failover:
++
+[source,terminal]
+----
+$ oc get deployment -l ipfailover --all-namespaces
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE   NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+default     ipfailover   2/2     2            2           105d
+----
+
+. Delete the deployment:
++
+[source,terminal]
+----
+$ oc delete deployment <ipfailover_deployment_name>
```

Comment:
oc delete deployment <ipfailover_deployment_name> -n default

as it not mentioned to change project.

Reply:
In cases where the customer can choose the namespace, the `-n <namespace>` part of the command is not shown and the documentation relies on the reference doc for the `oc` command.